jboss-eap-config-jgroups/module.yaml

```diff
@@ -0,0 +1,32 @@
+schema_version: 1
+name: jboss.eap.config.jgroups
+version: '1.0'
+description: JBoss EAP jgroups configuration
+execute:
+- script: configure.sh
+  user: '185'
+envs:
+    - name: "JGROUPS_ENCRYPT_PROTOCOL"
+      example: "SYM_ENCRYPT"
+      description: The JGroups protocol to use for encryption of the cluster traffic. Can be one of `SYM_ENCRYPT` (the default), or `ASYM_ENCRYPT`. If set to `SYM_ENCRYPT`, the definition of the JGroups `AUTH` protocol is optional, and can be performed by setting the **JGROUPS_CLUSTER_PASSWORD** environment variable. If set to `ASYM_ENCRYPT`, the definition of the JGroups `AUTH` protocol is **required**, and must be performed by setting the **JGROUPS_CLUSTER_PASSWORD** variable. The definition of the JGroups JCEKS keystore is **not** **expected**, when using the `ASYM_ENCRYPT` encryption protocol. A warning will be issued in this case, if such a keystore is defined.
+    - name: "JGROUPS_ENCRYPT_KEYSTORE"
+      example: "mykeystore.jceks"
+      description: "The JGroups keystore containing the key. If this does not begin with a directory separator (such as /) this is interpreted as a location relative to jboss.server.config.dir."
+    - name: "JGROUPS_ENCRYPT_PASSWORD"
+      example: "mypassword"
+      description: "The JGroups encryption password."
+    - name: "JGROUPS_ENCRYPT_KEYSTORE_TYPE"
+      example: "JCEKS"
```

Comment:
This example value should be the same as the elyton module one unless there's a reason the typical value would be different. Otherwise a careful reader would think there's a reason to use different values.

Reply:
OK, will update.